packages/app/src/contacts/components/contacts-select-modal/contacts-select-modal.component.tsx

```diff
@@ -0,0 +1,91 @@
+/**
+ * Copyright (c) Mudita sp. z o.o. All rights reserved.
+ * For licensing, see https://github.com/mudita/mudita-center/blob/master/LICENSE.md
+ */
+
+import React, { useState, useEffect } from "react"
+import { defineMessages } from "react-intl"
+import { useSelector } from "react-redux"
+import { contactHashSelector } from "App/contacts/selectors"
+import { createFullName } from "App/contacts/helpers/contacts.helpers"
+import { Contact } from "App/contacts/dto"
+import { intl } from "App/__deprecated__/renderer/utils/intl"
+import { FunctionComponent } from "App/__deprecated__/renderer/types/function-component.interface"
+import { ModalSize } from "App/__deprecated__/renderer/components/core/modal/modal.interface"
+import { ModalDialog } from "App/ui/components/modal-dialog"
+import { ContactsSelectModalProps } from "App/contacts/components/contacts-select-modal/contacts-select-modal.interface"
+import { ContactSimpleList } from "App/contacts/components/contact-simple-list"
+import { ContactInputSearch } from "App/contacts/components/contact-input-search"
+
+const messages = defineMessages({
+  title: { id: "module.messagesAttachModalTitle" },
+  unnamedContact: { id: "module.contacts.listUnnamedContact" },
+  emptyListTitle: { id: "module.contacts.emptyListTitle" },
+  emptySearchDescription: { id: "module.contacts.emptySearchDescription" },
+})
+
+export const isContactMatching = (
```

Comment:
1. Shouldn't it go to some external file like `is-contact-matching.ts`?
2. Seems to me that `contacts.component.tsx` contains exactly the same function - maybe those should use the same function?

Reply:
https://github.com/mudita/mudita-center/pull/815/commits/e106677ef12900f8f779bce6c3a2bf330a9f58f3